.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "cmake.configureOnOpen": false
+}
```

Comment:
Note: We don't need this IDE-specific config in the codebase. Can you remove it?

Reply:
Very sorry !   I've removed it.

Reply:
Thx.